### DISCUSSION TITLE
Making the author in Last Updated a link

### DISCUSSION THREAD

**Discussion by nightknighto** [facebook/docusaurus#8149] [Q&A]
Is it possible to configure the "last updated" section, to edit the name of the author to be a link pointing to his github page?

Also is there a place where I can have more in-depth sight into how "last updated" is compiled and how I can edit or modify it, if that's even possible?

**Comment by slorber** [comment]
Hi,

We cannot assume all users are using GitHub. Some are using Git(nothub), some are using GitLab, some are not even using version control. 

Code: https://github.com/facebook/docusaurus/blob/825211f41dc7eebca08516065e8cc7e33d99ab58/packages/docusaurus-plugin-content-docs/src/lastUpdate.ts

We read commit infos from Git, and it's not written in Git if the last commiter is from GitHub nor what is their GitHub profile. At most we may have a git email address to try to resolve to a profile, but how to make this resolution efficient and configurable? We don't have such feature atm.

If you want to provide such mapping (git commiter name => link) you'd have to maintain it yourself and swizzle the LastUpdated component to plug your own system

Note we have a `last_update` docs frontmatter (https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#last_update). It could be possible to add something like `last_update. author_url` frontMatter or something for convenience to set it manually
